daemon/meshnet/meshnet.go

```diff
@@ -79,18 +82,39 @@ func New(cfg Config) (*Meshnet, error) {
 	if err != nil {
 		return nil, err
 	}
+	// If the link type is GRPC then set the GRPC logging level to LevelNone
+	// Otherwise there will be GRPC log for every packet sent as for link type GRPC, GRPC is also the data-plane. This is too
+	// much of log that does not help in debugging and K8S does log rotation very frequently.
+	var svr *grpc.Server
+	lnkTyp := os.Getenv("INTER_NODE_LINK_TYPE")
+	if lnkTyp == "GRPC" { //+++rev: use the constant INTER_NODE_LINK_GRPC form plugin code. There need to be some shared constants between plugin and daemon.
+		svr = grpc.NewServer(cfg.GRPCOpts...)
+	} else {
+		svr = newServerWithLogging(cfg.GRPCOpts...)
+	}
+
 	m := &Meshnet{
 		config:  cfg,
 		rCfg:    rCfg,
 		kClient: kClient,
 		tClient: tClient,
 		lis:     lis,
-		s:       newServerWithLogging(cfg.GRPCOpts...),
+		s:       svr,
 	}
 	mpb.RegisterLocalServer(m.s, m)
 	mpb.RegisterRemoteServer(m.s, m)
 	mpb.RegisterWireProtocolServer(m.s, m)
 	reflection.Register(m.s)
+
+	// After server is registered, reduce logging if link type is GRPC
```

Comment:
what's the reason for this? would it make sense to just adhere to a global configuration option?

Reply:
When we use VxLan then GRPC is meshnet control plane and with the current option for every GRPC RCP execution it log for SUCESS, WRANING and ERROR. This is reasonable amount of log as there are no too many RPC call. 

With GRPC wire GRPCP also be come the data plane. Then for every packet exchanged between  pods, GRPC generates a SUCCESS log for each packet transfer.  Per packet logs are too much logging and the rolling logs over write the previous logs very quickly. As a results we loose helpful information that is needed for debugging. So for GRPC data plane we have stopped all logs for now. In the next submit we intend to enable only ERROR log. 

For vxLAN it remains as is.